### DISCUSSION TITLE
expand Dimensions in html_repr of a dataset?

### DISCUSSION THREAD

**Discussion by raybellwaves** [pydata/xarray#5004] [Q&A]
Am I right in thinking you can't expand/contract the Dimensions part of a html repr of a dataset? Haven't generated an object with 100 dims to test properly

```
url = 'https://thredds.ucar.edu/thredds/dodsC/grib/NCEP/GFS/Global_0p25deg/Best'
xr.open_dataset(url)
```

![Screen Shot 2021-03-05 at 1 13 52 PM](https://user-images.githubusercontent.com/17162724/110157343-0ec14880-7db6-11eb-8300-b7c927fa7e9d.png)

**Comment by max-sixty** [comment]
That's a lot of dimensions already! 😄 

Assuming we never shorten Dimensions, is the change to remove the "arrow" / triangle? Or at least have it default point downwards?

**Reply by shoyer** [reply]
Ouch, this is indeed a lot of dimensions. I agree that support for contracting dimensions in the repr would seem useful for cases like this.
